src/scripts/validate.sh

```diff
@@ -6,4 +6,14 @@ if [ "https://circleci.com" != "${CIRCLECI_API_HOST}" ] && [ -z "${CIRCLE_TOKEN}
     exit 1
 fi
 
-circleci orb validate --host "${CIRCLECI_API_HOST:-https://circleci.com}" --token "${CIRCLE_TOKEN:-dummy}" --skip-update-check "${ORB_PARAM_OUTPUT_DIR}orb.yml"
+if [ -n "${CIRCLE_TOKEN}" ]; then
```

Comment:
This is using the internal CIRCLE_TOKEN, which is a good default in case there is no token provided, but I see that the "CIRCLECI_TOKEN" parameter is unused.